### PR TITLE
Feature#8#9/style the sign in and sign up view

### LINKED ISSUE #8: Style the Sign In view.
# REQUIREMENTS
Add the sass styles to the sign in view.

# DESIGN
image
# Acceptance Criteria

List all required acceptance criteria
- [x] Create the buttons
- [x] Create the inputs
- [x] Use the Figma design to style the view. #2

### LINKED ISSUE #9: Stlye the Sign Up view.
# REQUIREMENTS
Style the design from the Figma design.

# DESIGN
image

# Acceptance Criteria
- [ ] Use the buttons created in the #8 issue
- [ ] Use the inputs created in the #8 issue
- [ ] Style the view based in the Figma design. #2

### PR DESCRIPTION
**Pull request by EliasUR**
# Description

 - What?: I've added a responsive design to sign in and sign up view.
 - Why?: To improve user experience on both desktop and mobile.
 
  ## Screenshots 
- Desktop

image

image

- Mobile

image

image

Closes #8 
Closes #9

### CODE REVIEW

**Review by AlejandroCuriel — APPROVED** [review]
LGTM :rocket: :guitar:

**Review by germanruzca — APPROVED** [review]
LGTM, looks great! 🚀

File: app/assets/stylesheets/components/_logo.scss [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  width: 70px;
+  
+  h2{
+    color: $gray;
+    text-shadow: black 1px 0px 0px, black 0.540302px 0.841471px 0px,
+              black -0.416147px 0.909297px 0px,black -0.989992px 0.14112px 0px, 
+              black -0.653644px -0.756802px 0px, black 0.283662px -0.958924px 0px,
+              black 0.96017px -0.279415px 0px;
+  }
+}
```
Comment (AlejandroCuriel) [inline app/assets/stylesheets/components/_logo.scss:19]: add an empty space

File: app/assets/stylesheets/application.scss [hunk 2]
```diff
@@ -1,10 +1,16 @@
...
 @import 'utils/variables';
 @import 'base/globals';
 @import 'components/buttons';
+@import 'components/logo';
+@import 'components/session_form';
 @import 'layout/header';
 @import 'layout/footer';
 @import 'utils/helpers';
+
+main{
```
Comment (AlejandroCuriel) [inline app/assets/stylesheets/application.scss:14]: space between name and brace:
example:
.main {
}

File: app/assets/stylesheets/components/_buttons.scss [hunk 3]
```diff
@@ -0,0 +1,9 @@
+.btn-primary{
```
Comment (AlejandroCuriel) [inline app/assets/stylesheets/components/_buttons.scss:1]: space between name and brace:
example:
.btn-primary {
}

File: app/assets/stylesheets/components/_buttons.scss [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+  background-color: $red;
+  font-weight: 700;
+  font-size: large;
+
+  &:hover, &:active{
```
Comment (AlejandroCuriel) [inline app/assets/stylesheets/components/_buttons.scss:6]: space between name and brace:
example:
&:hover, &:active {
}

File: app/assets/stylesheets/components/_logo.scss [hunk 5]
```diff
@@ -0,0 +1,19 @@
+.logo{
```
Comment (AlejandroCuriel) [inline app/assets/stylesheets/components/_logo.scss:1]: fix all tags that do not have a space between the name and the brace

File: app/assets/stylesheets/components/_session_form.scss [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+      }
+    }
+
+    @media (orientation:landscape){
+      width: 30%;
+    }
+  }
+
+  .field{
+    margin: 3% 0% 3% 0%;
```
Comment (AlejandroCuriel) [inline app/assets/stylesheets/components/_session_form.scss:32]: you can simplify this as follows: margin: 3% 0;
Here you can find more information: https://developer.mozilla.org/es/docs/Web/CSS/margin

**Review by AlejandroCuriel — CHANGES_REQUESTED** [review]
fix some small style details <3

File: app/assets/stylesheets/components/_session_form.scss [hunk 7]
```diff
@@ -0,0 +1,39 @@
+.session_form{
+  align-items: center;
+  display: flex;
+  font-weight: 600;
+  height: 100%;
+  justify-content: center;
+  width: 100%;
+
+  &__card{
```
Comment (JuanequeX) [inline app/assets/stylesheets/components/_session_form.scss:9]: Add a extra space between your clasname and yout bracket

File: app/assets/stylesheets/components/_session_form.scss [hunk 8]
```diff
@@ -0,0 +1,39 @@
+.session_form{
```
Comment (JuanequeX) [inline app/assets/stylesheets/components/_session_form.scss:1]: Add a extra space between your clasname and yout bracket

File: app/assets/stylesheets/components/_session_form.scss [hunk 9]
```diff
@@ -0,0 +1,39 @@
...
+  align-items: center;
+  display: flex;
+  font-weight: 600;
+  height: 100%;
+  justify-content: center;
+  width: 100%;
+
+  &__card{
+    background-color: white;
```
Comment (JuanequeX) [inline app/assets/stylesheets/components/_session_form.scss:10]: Use variables to add color, can made a specific file for it, could be name it variables.scss

File: app/assets/stylesheets/components/_session_form.scss [hunk 10]
```diff
@@ -0,0 +1,39 @@
...
+    border: 1px solid #00000020;
```
Comment (JuanequeX) [inline app/assets/stylesheets/components/_session_form.scss:11]: Same observation here with the color.

File: app/assets/stylesheets/components/_session_form.scss [hunk 11]
```diff
@@ -0,0 +1,39 @@
...
+    border-radius: 3%;
+    margin: 5%;
+    padding: 3%;
+    
+    &__link{
```
Comment (JuanequeX) [inline app/assets/stylesheets/components/_session_form.scss:16]: Add a extra space between your clasname and yout bracket

File: app/assets/stylesheets/components/_session_form.scss [hunk 12]
```diff
@@ -0,0 +1,39 @@
...
+      color: $text-gray;
+      font-size: 15px;
+      text-decoration: none;
+      
+      &:hover{
```
Comment (JuanequeX) [inline app/assets/stylesheets/components/_session_form.scss:21]: Add a extra space between your clasname and yout bracket

File: app/assets/stylesheets/components/_session_form.scss [hunk 13]
```diff
@@ -0,0 +1,39 @@
...
+        color: $active-red;
+      }
+    }
+
+    @media (orientation:landscape){
```
Comment (JuanequeX) [inline app/assets/stylesheets/components/_session_form.scss:26]: Add a extra space between your clasname and yout bracket

File: app/assets/stylesheets/components/_session_form.scss [hunk 14]
```diff
@@ -0,0 +1,39 @@
...
+      width: 30%;
+    }
+  }
+
+  .field{
```
Comment (JuanequeX) [inline app/assets/stylesheets/components/_session_form.scss:31]: Add a extra space between your clasname and yout bracket

File: app/assets/stylesheets/components/_session_form.scss [hunk 15]
```diff
@@ -0,0 +1,39 @@
...
+    margin: 3% 0% 3% 0%;
+
+    .form-control{
```
Comment (JuanequeX) [inline app/assets/stylesheets/components/_session_form.scss:34]: Add a extra space between your clasname and yout bracket

**Review by JuanequeX — CHANGES_REQUESTED** [review]

File: app/assets/stylesheets/components/_session_form.scss (hunk 10 above)
Comment (EliasUR) [reply app/assets/stylesheets/components/_session_form.scss]: Thanks, I've added this variables

**Review by germanruzca — DISMISSED** [review]
LGTM

**Review by AlejandroCuriel — APPROVED** [review]
LGTM CRACK! :rocket:

**Review by YilKanda — CHANGES_REQUESTED** [review]
Update your branch with respect to master.

**Review by VictorElizalde — APPROVED** [review]
LGTM!

**Pull request merged**